src/features/fwiCalculator/components/IOSSlider.tsx

```diff
@@ -0,0 +1,59 @@
+import { Slider, withStyles } from '@material-ui/core'
+import theme from 'app/theme'
+
+const iOSBoxShadow =
+  '0 3px 1px rgba(0,0,0,0.1),0 4px 8px rgba(0,0,0,0.13),0 0 0 1px rgba(0,0,0,0.02)'
+
+const IOSSlider = withStyles({
```

Comment:
Give it a more appropriate name - and as per @Kyubinhan move to generic components folder.

Reply:
Renamed the component to WpsSlider.